frontend/src/Context/WebSocketContext.jsx

```diff
@@ -9,7 +9,7 @@ export const WebSocketProvider = ({ children }) => {
     const [error, setError] = useState(null);
 
     useEffect(() => {
-        const ws = new WebSocket('wss://co2-api.duckdns.org');
+        const ws = new WebSocket('wss://third-year-studio-2024-fresh-air-backend.onrender.com');
```

Comment:
This seems to be on @declandeh00 PR as well, this should still be suing the co2-api url rather than render due to the render DB no longer being active.